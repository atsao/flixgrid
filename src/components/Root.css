body {
  font-family: Helvetica Neue, Arial, sans-serif;
  font-size: 14px;
}

.wrapper {
  margin: 0 auto;
  width: 90%
}

@media (max-width: 568px) {
  .wrapper {
    overflow: scroll;
    min-width: 48em;
  }
}
