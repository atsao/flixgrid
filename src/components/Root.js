import React, { Component } from 'react';
import PropTypes from 'prop-types';

import styles from '../assets/css/index.css';

import Flixgrid from './Flixgrid';

class Root extends Component {
  render() {
    return (
      <div className={styles['wrapper']}>
        <Flixgrid />
      </div>
    );
  }
}

Root.propTypes = {
  children: PropTypes.node,
};

export default Root;
