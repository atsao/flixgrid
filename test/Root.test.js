import React from 'react';
import renderer from 'react-test-renderer';

import Root from '../src/components/Root';

it('renders a Root component', () => {
  const tree = renderer.create(<Root />);
  expect(tree).toMatchSnapshot();
});
