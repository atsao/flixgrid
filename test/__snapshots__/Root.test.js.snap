// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Root component 1`] = `
<div
  className="wrapper"
>
  <table>
    <tbody>
      <tr>
        <td>
          nothing
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
